packages/airnode-node/src/api/processing.ts

```diff
@@ -75,21 +78,29 @@ export const preProcessApiSpecifications = async (payload: ApiCallPayload): Prom
   } as ApiCallPayload;
 };
 
-export const postProcessApiSpecifications = async (input: unknown, endpoint: Endpoint) => {
+export const postProcessApiSpecifications = async (input: unknown, endpoint: Endpoint, payload: ApiCallPayload) => {
   const { postProcessingSpecifications } = endpoint;
 
   if (!postProcessingSpecifications || postProcessingSpecifications?.length === 0) {
     return input;
   }
 
+  // provide endpoint parameters immutably
+  let endpointParameters = JSON.parse(JSON.stringify(payload.aggregatedApiCall.parameters));
+
+  // The HTTP gateway is a special case for ChainAPI that is not allowed to access reserved parameters
```

Comment:
> We want both pre-processing and post-processing to not have access to the reserved parameters

OK, I've removed access to reserved parameters from pre- and post-processing.

> This should apply to all cases, and not only calls coming through the gateway.

The title and description of #1738 was specific for the gateway (because of ChainAPI test calls) and so that is what I addressed in #1794. But easy enough and makes sense. I've made the change.

Reply:
Right, I now realize that that is about an older PR. @andreogle for visibility I guess :sweat_smile: The fix looks good, thanks.